setup.py

```diff
@@ -100,6 +100,7 @@ def extra(self):
     extras_require={
         'ujson': ['ujson>=1.35'],
         'email': ['email-validator>=1.0.3'],
+        'typing_extensions': ['typing-extensions>=3.7.2']
```

Comment:
I'm not sure what the key should be. Obviously it's `import typing_extensions` in code, but the package can be installed via `pip install typing-extensions` or `pip install typing_extensions`

Reply:
I think this is fine as long as it's documented.